decidim-blogs/spec/forms/decidim/blogs/admin/post_form_spec.rb

```diff
@@ -52,15 +54,35 @@ module Admin
             subject.map_model(post)
           end
 
-          it "does not assign the user group for normal users" do
-            expect(subject.user_group_id).to be_nil
+          it "assigns current_user as user_group_id" do
+            expect(subject.user_group_id).to eq("current_user")
+          end
+
+          it "assigns user name as author" do
+            expect(subject.author).to be_nil
```

Comment:
```suggestion
            expect(subject.author).to eq(current_organization)
```
Following the same logic, the default status for `author_id` is "current_organization", so this method should return the organization here

Reply:
Done.